code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -69,24 +73,38 @@ GLOBAL_LIST_INIT(sandbag_recipes, list ( \
 	. = ..()
 	. += GLOB.sandbag_recipes
 
-/obj/item/emptysandbag
-	name = "empty sandbag"
+/obj/item/stack/sheet/mineral/emptysandbags
+	name = "empty sandbags"
+	singular_name = "empty sandbag"
 	desc = "A bag to be filled with sand."
 	icon = 'icons/obj/stack_objects.dmi'
-	icon_state = "sandbag"
+	icon_state = "sandbag-stack"
+	inhand_icon_state = "sandbag-stack"
+	drop_sound = 'sound/items/handling/cloth_drop.ogg'
+	pickup_sound = 'sound/items/handling/cloth_pickup.ogg'
 	w_class = WEIGHT_CLASS_TINY
 
-/obj/item/emptysandbag/attackby(obj/item/W, mob/user, params)
+/obj/item/stack/sheet/mineral/emptysandbags/fifty
+	amount = 50
+
+/obj/item/stack/sheet/mineral/emptysandbags/attackby(obj/item/W, mob/user, params)
+	add_fingerprint(user)
 	if(istype(W, /obj/item/stack/ore/glass))
```

Comment:
early return please (check the inverse of this and call return ..() to avoid indentation)